src/scss/config/animation/_index.scss

```diff
@@ -0,0 +1,42 @@
+@use '~sassdoc-theme-herman/scss/utilities' as herman;
+@use '../tools';
+@use 'changes';
+@use 'times';
+@use 'sass:meta';
+@forward 'changes';
+@forward 'times';
+
+/// # Animation Config
```

Comment:
We don't need `@include herman.add ...` unless we are wanting Herman to show a visual preview, right?